src/legacy/ui/public/agg_types/controls/field.tsx

```diff
@@ -23,9 +23,8 @@ import React, { useEffect } from 'react';
 import { EuiComboBox, EuiComboBoxOptionProps, EuiFormRow } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { AggConfig } from 'ui/vis';
+import { AggParamEditorProps, ComboBoxGroupedOption } from 'ui/vis/editors/default';
```

Comment:
The latest news I heard from @alexwizp , we try to get rid of such aliases as `ui/...`